FridgeExercise/Shelf.cs

```diff
@@ -0,0 +1,35 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Diagnostics.Metrics;
+using System.Drawing;
+using System.Linq;
+using System.Reflection;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace FridgeExercise
+{
+	internal class Shelf
+	{
+		public int ShelfID { get; set; }
+		private int CounterID = 1;
+		public int ShelfsFloor { get; set; }
+		public double SpaceInShelf { get; set; }
```

Comment:
```suggestion
		public double AvailableSpaceInSquareMeters{ get; set; }
```
If you have a better name in mind that's descriptive but not too long, obviously use it :) That's just a suggestion